gravitee-apim-console-webui/src/shared/components/specific-json-schema-type/resource-type.component.html

```diff
@@ -35,6 +35,10 @@
   <mat-hint *ngIf="props.description || props.hintStart as hint">
     {{ hint }}
   </mat-hint>
+
+  <mat-hint *ngIf="resourceNotExist$ | async">
+    Resource doesn't exist. Please take care to add it with the API so that it is taken into account.
```

Comment:
`does not`
@jourdiw should we prefer "long" form ?

Reply:
```suggestion
    Resource does not exist. Please add it to the API for it to be taken into account.
```

Reply:
~~censored joke~~

🚀